unix/qstrdefsport.h

```diff
@@ -79,6 +79,7 @@ Q(listen)
 Q(accept)
 Q(recv)
 Q(recvfrom)
+Q(settimeout)
```

Comment:
Then what should be done about this? As far as I can tell, I can remove a decent portion of the lines in this file and building still succeeds. I have no idea of what the implications of that are on the final binaries. So adding an ifdef for a single line seems silly to me when some major cleanup is needed here.